test/Microsoft.AspNetCore.Mvc.FunctionalTests/TagHelperSampleTest.cs

```diff
@@ -49,6 +65,34 @@ public TagHelperSampleTest(MvcTestFixture<TagHelperSample.Web.Startup> fixture)
 #endif
         }
 
+        [Fact]
+        public async Task HomeController_Index_ReturnsExpectedContent_AfterReset()
+        {
+            // Arrange
+            var resetResponse = await Client.PostAsync("http://localhost/Home/Reset", content: null);
```

Comment:
Refactor this and next line into a helper method for future tests. The pattern is used a few times in the file now.